packages/edit-post/src/components/sync-state-with-url/use-init-edited-entity-from-url.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * WordPress dependencies
+ */
+import { privateApis as routerPrivateApis } from '@wordpress/router';
+
+/**
+ * Internal dependencies
+ */
+import { unlock } from '../../lock-unlock';
+
+const { useLocation } = unlock( routerPrivateApis );
+
+export default function useInitEditedEntityFromURL() {
+	const { params = {} } = useLocation();
```

Comment:
Do we want to introduce the routing into the post editor? This is a new public API btw. (the URLs)
I initially though, we could just keep some internal state with a list of edited entities.

Reply:
I was trying to set it up in a way that the browser back button worked as expected. If just managing with internal state that won't be the case, unless I am missing something. Happy to set it up without routing if the user back button experience is not an issue.